sonar-plugin-api/src/main/java/org/sonar/api/server/ws/internal/ValidatingRequest.java

```diff
@@ -139,9 +151,33 @@ private String readParamOrDefaultValue(String key, @Nullable WebService.Param de
     return value;
   }
 
+  private List<String> readMultiParamOrDefaultValue(String key, @Nullable WebService.Param definition) {
+    if (definition == null) {
+      String message = String.format("BUG - parameter '%s' is undefined for action '%s'", key, action.key());
+      Loggers.get(getClass()).error(message);
```

Comment:
I did as done in other similar readParam. The behavior should be consistent, and I didn't have idea of impacts